packages/eas-cli/src/utils/json.ts

```diff
@@ -1,39 +0,0 @@
-import assert from 'assert';
-
-import Log from '../log';
-
-let stdoutWrite: NodeJS.WriteStream['write'] | undefined;
-
-export function enableJsonOutput(): void {
-  if (stdoutWrite) {
-    return;
-  }
-  stdoutWrite = process.stdout.write;
-  process.stdout.write = process.stderr.write.bind(process.stderr);
```

Comment:
As far as I can see you are not handling a case where spawned process writes to stdout, that output also needs to be redirected in stderr in JSON only mode.